Dockerfile

```diff
@@ -1,23 +1,38 @@
-FROM node:11-alpine
+FROM ubuntu:16.04
 
 # minimal apk dependencies to be safe
-ENV PACKAGES="curl ca-certificates" \
-    NODE_ENV="production" \
-    HOST="0.0.0.0:5000"
+ENV PACKAGES="ca-certificates git redis-server nginx software-properties-common python-software-properties nodejs"
+ENV NODE_ENV="production"
+ENV HOST="0.0.0.0:5001"
 
 WORKDIR /usr/src/app
 
 COPY package*.json yarn.lock ./
+RUN apt-get update
+RUN apt-get install -y curl sudo
+RUN curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash
 
-RUN apk add --no-cache $PACKAGES && \
-    yarn --prod
+RUN apt-get install -y ${PACKAGES}
+RUN npm i -g yarn
+RUN yarn --prod
 
-COPY . .
+RUN add-apt-repository universe
+RUN add-apt-repository ppa:certbot/certbot
+RUN apt-get update
+RUN apt-get install certbot python-certbot-nginx  -y
+
+COPY build build
```

Comment:
Instead of using the current `build:docker` script which builds outside of the container and copies it, let's build it within this script or perhaps on the entrypoint.

Reply:
Can easily do it, but  why? It will require to install `--dev` deps in Yarn. See no reason. Just explain me your intention please.

Reply:
This way you can run the Docker setup without having to install dependencies outside. By containerizing the build as well, we ensure the build and its dependencies are also up-to-date.